### PR TITLE
Add rulesets docs for `renovate`

### PR DESCRIPTION
**Pull request by paddyroddy**

### CODE REVIEW

**Review by samcunliffe — COMMENTED** [review]
All good, but I'd err towards using the renovate names for things.
[GitHub rulesets](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/about-rulesets#about-rulesets) are a thing, but renovate seems to just call them "Configuration options".

**Comment by paddyroddy** [comment]
> All good, but I'd err towards using the renovate names for things. [GitHub rulesets](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/about-rulesets#about-rulesets) are a thing, but renovate seems to just call them "Configuration options".

Considering that, you have to add `rulsets` to GitHub (AKA nothing to do with `renovate`). I don't really follow.

**Comment by samcunliffe** [comment]
> `rulsets` to GitHub (AKA nothing to do with `renovate`). I don't really follow.

Ah then it's just my confusion. You started talking about rulesets in renovate/README.md and it wasn't clear what you meant.

File: renovate/README.md [hunk 1]
```diff
@@ -47,3 +49,89 @@ following.
 
 - `Allow specified actors to bypass required pull requests` ✅
   - Add `renovate` here
+
+### Rulesets
+
+Alternatively, if using rulesets you can modify and import the following two
+rulesets.
```
Comment (samcunliffe) [inline renovate/README.md:56]:
```suggestion
Alternatively, if you're using [GitHub rulesets](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/about-rulesets) in your repo, you can modify and import the following two.
```

**Review by samcunliffe — APPROVED** [review]

**Comment by paddyroddy** [comment]
> Ah then it's just my confusion. You started talking about rulesets in renovate/README.md and it wasn't clear what you meant.

To be fair, if you read the whole README, I talk about branch protection & then rulesets.

**Pull request merged**